engine/runtime/src/main/java/org/enso/interpreter/EnsoLanguage.java

```diff
@@ -337,4 +337,13 @@ protected Object getScope(EnsoContext context) {
   public Optional<IdExecutionService> getIdExecutionService() {
     return idExecutionInstrument;
   }
+
+  /** Conversions of primitive values */
+  protected Object getLanguageView(EnsoContext context, Object value) {
+    if (value instanceof Boolean b ) {
```

Comment:
We don't need conversions for other values?

Reply:
Not yet. I would need one for `Decimal`, but values of decimal are `double`, so there is nothing to convert. Other than that there was no need to convert anything else that `Boolean`.